tests/project/settings.py

```diff
@@ -77,6 +77,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "allauth.account.middleware.AccountMiddleware",
```

Comment:
also needs to be added in the `INSTALLED_APPS` as `allauth`, the tests indicate the error.
@goapunk